src/components/accordion-card/accordion-card.js

```diff
@@ -0,0 +1,73 @@
+import React, { Component } from 'react';
+import cx from 'classnames';
+import PropTypes from 'prop-types';
+import Button from 'components/button';
+import Icon from 'components/icon';
+import chevronIcon from 'assets/icons/chevron.svg';
+import styles from './accordion-card-styles.scss';
+
+class AccordionCard extends Component {
+  constructor(props) {
+    super(props);
+    this.state = { isOpen: false };
```

Comment:
Why don't initialise the state directly here instead of the componentDidMount? we will avoid a render

Reply:
⚡️